kolibri/core/content/contentschema/versions/content_schema_current.py

```diff
@@ -45,6 +46,9 @@ class ContentLocalfile(Base):
 class ContentContentnode(Base):
     __tablename__ = "content_contentnode"
     __table_args__ = (
+        CheckConstraint(
```

Comment:
Adding the constraint checks seems to be causing the Python test errors.

Reply:
Fixed